man/git-mv-remote.md

```diff
@@ -0,0 +1,44 @@
+git-mv-remote(1) -- Rename a remote
```

Comment:
What about changing the name to `git-rename-remote`? Since we already have `git-rename-branch` and `git-rename-tag`.

Reply:
done